packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -57,6 +75,14 @@ void main(List<String> testRunnerArgs) async {
   }
 }
 
+bool _isUnsupportedForDevice(String testPath, {required String testAppDevice}) {
```

Comment:
can we rename this method to be worded in positive tense `isSupportedForDevice`? the double negatives make this a bit hard to follow

Reply:
Done, also changed to a method on the `TestAppDevice` enhanced enum